app/src/main/java/com/github/epfl/meili/map/MapActivity.kt

```diff
@@ -99,14 +99,11 @@ class MapActivity : AppCompatActivity(), OnMapReadyCallback {
 
         // Add on click listener
         clusterManager.setOnClusterItemClickListener {
-            val intent = Intent(this, PoiActivity::class.java)
```

Comment:
Why did you remove this part? This was supposed to launch the PoiInfo activity when you click on a Poi

Reply:
I removed it because the query made in the poi info activity was not working (it didn't return anything). So I think that something doesn't work in the PointOfInterest which is passed in the intent by the MapActivity.

Reply:
I see that's probably because the uid of the poi is from the overpass api and not from Google Places API. This will be fixed in my next PR. I think we should still keep it and the activity should be failure-safe and handle the case when the response is empty. What do you think?

Reply:
Yes, that's better. It's done now.

Reply:
Perfect! Thanks for the work